test/test_tensorexpr.py

```diff
@@ -1260,5 +1260,18 @@ def test(x):
         assert torch.allclose(scripted(a), 2 * a)
         assert cx.elapsed_value() == 1
 
+    def test_simple_add(self):
+        torch._C._jit_set_te_generate_block_code(True)
```

Comment:
The standard practice is to remember the old value before, and only revert to that when you are done.